event/handler.go

```diff
@@ -109,28 +130,31 @@ func (handler *ExportHandler) Handle(event *FilterSubmitted) error {
 	return nil
 }
 
-func (handler *ExportHandler) getVersionState(event *FilterSubmitted) (string, error) {
-	// We currently only get a filter id for a filter job so return published
-	// TODO: ensure an instance id is given for filter jobs
-	if len(event.FilterID) > 0 {
-		return "published", nil
+func (handler *ExportHandler) getFilterOutputState(event *FilterSubmitted) (bool, error) {
```

Comment:
The function name should be updated because the return types don't match the function name